Services/CO.CDP.OrganisationInformation.Persistence.Tests/DatabaseAuthenticationKeyRepositoryTest.cs

```diff
@@ -76,27 +76,27 @@ public void ItRejectsTwoApiKeyWithTheSameName()
         using var repository = AuthenticationKeyRepository();
         var organisation = GivenOrganisation();
         var key = Guid.NewGuid().ToString();
-        var authenticationKey1 = GivenAuthenticationKey(key: key, organisation: organisation);
-        var authenticationKey2 = GivenAuthenticationKey(key: key, organisation: organisation);
+        var authenticationKey1 = GivenAuthenticationKey(name: "fts", key: key, organisation: organisation);
+        var authenticationKey2 = GivenAuthenticationKey(name: "fts", key: key, organisation: organisation);
 
         repository.Save(authenticationKey1);
-        
-        repository.Invoking(r => r.Save(authenticationKey2))
+
+        repository.Invoking(async r => await r.Save(authenticationKey2))
             .Should().ThrowAsync<IAuthenticationKeyRepository.AuthenticationKeyRepositoryException.DuplicateAuthenticationKeyNameException>()
             .WithMessage($"Authentication Key with name `fts` already exists.");
     }
 
     private static AuthenticationKey GivenAuthenticationKey(
-        string name = "fts",
-        string key = "api-key",
+        string? name = null,
+        string? key = null,
         Organisation? organisation = null,
         List<string>? scopes = null
     )
     {
         return new AuthenticationKey
         {
-            Name = name,
-            Key = key,
+            Name = name ?? $"key {Guid.NewGuid()}",
+            Key = key ?? Guid.NewGuid().ToString(),
```

Comment:
I randomised details, so it's:
*  unlikely two tests create the same entry
* person writing the next test relies on generated details but will pass them in instead